crates/pubsub/src/lib.rs

```diff
@@ -206,6 +218,7 @@ impl SubInner {
 #[derive(Debug, Clone)]
 enum PubImplementor {
     ConfluentApacheKafka(PubConfluentApacheKafkaImplementor),
+    Local(MosquittoImplementor),
```

Comment:
Why is it called Local instead of Mosquitto? "Local", to me, means that the implementor is purely relying on the underlying OS, like the OS File System, or OS Devices. I don't think Mosquitto is considered to be a OS component.

Reply:
I'll change this.

Reply:
done ✅